esp8266-fastled-webserver/data/index.htm

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>ESP8266 + FastLED by Evil Genius Labs</title>
+  <title>&nbsp;</title>
```

Comment:
SIMPLE FILE REVIEW

Changes to support dynamically modifying the product name in title and elements.  See also `app.js`.

Also adds template for "section header", which makes the settings page make a **_whole_** lot more sense.